### DISCUSSION TITLE
Setting Menu on windows 10 results in the app just hanging at fetching dependencies.

### DISCUSSION THREAD

**Discussion by jeremiahsherrill** [NativePHP/laravel#387] [Q&A]
Windows 10 64 bit
"laravel/framework": "^10.10",
"nativephp/electron": "^0.7.3"

```
public function boot(): void
    {
        Menu::new()
            ->appMenu()
            ->submenu('NativePHP', Menu::new()
                ->link('https://nativephp.com', 'Documentation')
            )
            ->register();



        Window::open();
    }
```

Adding my own menu causes the application to hang, removing the menu results in the app starting. 

no errors being generated, it just locks up

Electron is running with very high power usage and about 12% of the cpu

**Comment by simonhamp** [comment]
Are there any errors in you Laravel logs?
